recipes-wpe/wpeframework/wpeframework-plugins-rdk_git.bb

```diff
@@ -3,61 +3,43 @@ LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=39fb5e7bc6aded7b6d2a5f5aa553425f"
 PR = "r1"
 
-require include/wpeframework-plugins.inc
-
-SRC_URI =  "git://git@github.com:/WebPlatformForEmbedded/ThunderNanoServicesRDK.git;protocol=ssh;branch=master"
-SRCREV = "f086d57f464c83f272d1ce9c1598612f1e8f6c40"
+SRC_URI = "git://git@github.com:/WebPlatformForEmbedded/ThunderNanoServicesRDK.git;protocol=ssh;branch=master"
+SRCREV = "aefc318b470b41ad8ea0858c01292970f1e8e8af"
 
 # ----------------------------------------------------------------------------
 
-# More complicated plugins are moved seperate includes
-
+# Plugins configs are moved to seperate includes
+require include/deviceidentification.inc
+require include/deviceinfo.inc
+require include/locationsync.inc
+require include/messenger.inc
+include include/monitor.inc
```

Comment:
Let's go with 'require' instead of include everywhere. The other option is to have include on plugins and require on commons, but for now I would go with version 1 as it's easier to debug